calculators/candidate_contributions_by_type.rb

```diff
@@ -50,34 +50,34 @@ def contributions_by_candidate_by_type
       # Schedule A during a preprocssing script. (See
       # `./../remove_duplicate_transactionts.sh`)
       monetary_results = ActiveRecord::Base.connection.execute <<-SQL
+        WITH combined_contributions AS (
+          SELECT "Filer_ID"::varchar, "Entity_Cd", "Tran_Amt1", "Tran_NamF", "Tran_NamL"
+          FROM "A-Contributions"
+          UNION ALL
+          SELECT "Filer_ID"::varchar, "Entity_Cd", "Tran_Amt1", "Tran_NamF", "Tran_NamL"
+          FROM "C-Contributions"
+          UNION ALL
+          SELECT
+            "Filer_ID"::varchar,
+            "Entity_Cd",
+            "Amount" as "Tran_Amt1",
+            "Enty_NamF" as "Tran_NamF",
+            "Enty_NamL" as "Tran_NamL"
+          FROM "497"
+          WHERE "Form_Type" = 'F497P1'
+        )
         SELECT
           "Filer_ID",
           CASE
             WHEN "FPPC" IS NULL THEN "Entity_Cd"
             ELSE 'SLF'
           END AS "Cd",
           SUM("Tran_Amt1") AS "Total"
-        FROM
-          (
-            SELECT "Filer_ID"::varchar, "Entity_Cd", "Tran_Amt1", "Tran_NamF", "Tran_NamL"
-            FROM "A-Contributions"
-            UNION ALL
-            SELECT "Filer_ID"::varchar, "Entity_Cd", "Tran_Amt1", "Tran_NamF", "Tran_NamL"
-            FROM "C-Contributions"
-            UNION ALL
-            SELECT "Filer_ID"::varchar, "Entity_Cd",
-              "Amount" as "Tran_Amt1",
-              "Enty_NamF" as "Tran_NamF",
-              "Enty_NamL" as "Tran_NamL"
-            FROM "497"
-            WHERE "Form_Type" = 'F497P1'
-          ) AS U
-          LEFT OUTER JOIN
-          "oakland_candidates"
-            ON  "FPPC"::varchar = "Filer_ID"
-              AND (LOWER("Candidate") = LOWER(CONCAT("Tran_NamF", ' ', "Tran_NamL"))
-                OR LOWER("Aliases") like
-                    LOWER(CONCAT('%', "Tran_NamF", ' ', "Tran_NamL", '%')))
+        FROM combined_contributions
+        LEFT OUTER JOIN "oakland_candidates"
```

Comment:
Perhaps a comment:
Identify contributions from the candidate to their committee by the name they used in the contribution.  It is not always exactly the same name.  "Aliases" are added as these are identified.

Reply:
Okay, I'll add a comment to that effect!